Jenkinsfile

```diff
@@ -0,0 +1,87 @@
+// To run this Jenkinsfile, you need to have the following plugins installed:
+// - NodeJS
+// - CloudBees AWS Credentials
+// - Pipeline: AWS Steps
+pipeline {
+    agent any
+    tools {nodejs "19.4"}
```

Comment:
Why not stick with an LTS?

Reply:
This was mostly for testing. We can switch out the version when we set up the Jenkins server